src/FoKL/FoKLRoutines.py

```diff
@@ -356,6 +813,9 @@ def gibbs(inputs, data, phis, Xin, discmtx, a, b, atau, btau, draws):
                 Lamb_tausqd_inv = np.diag(1 / np.diag(Lamb_tausqd))
 
                 mun = Q.dot(Lamb_tausqd_inv).dot(np.transpose(Q)).dot(Xty)
+                if mun.ndim == 1: # if mun.shape == (number,) != (number,1), then add new axis
+                    mun = mun[:, np.newaxis]
+                    warnings.warn("'mun' was made into (n,1) column vector from single list (n,). It is unclear why this was not already the case.",category=UserWarning)

```

Comment:
Line 818 - How often does this happen? Is it unclear why this is the case?

Reply:
does not happen with Sigmoid, GP Int, or F16

it could be deleted but might be good to keep in case of future de-bugging --- was likely originating from an old problem in auto_cleanData() which is now resolved

-------------------

what does happen is:
      (which isn't a problem but just a note for developing, so could be deleted from Routines or just import warnings and turn off UserWarnings)

Sigmoid ---> 'inputs' was transposed, ignore is more datapoints than input variables

GP Int -----> 'data' was made into (n,1) column vector ...

F16 ---> 'inputs' transposed, ignore .. ; 'data' transposed, ignore ...

Reply:
personally i like the "inputs/data was transposed, ignore if datapoints > variables" cause just in case someone tries to fit 10 inputs variables x 9 datapoints/timepoints, it'll let them know that their data was flipped

the "(n,) to (n,1)" is kinda pointless practically but i'm not 100% why it happens so might be worth keeping